.env

```diff
@@ -12,12 +12,7 @@ HASH_OPENSEARCH_PASSWORD=admin
 HASH_OPENSEARCH_PORT=9200
 HASH_OPENSEARCH_USERNAME=admin
 
-HASH_GRAPH_API_HOST=localhost
```

Comment:
**Blocking:** This is currently being used by the workspace backend and the workspace integration tests to connect to the Graph API. Was this accidentally removed?

Reply:
Yes, that was by accident (`POSTGRES_HOST` was on purpose as you shouldn't need to access `postgres` manually and the Dockerfile uses the network name instead.)
a2a18148040e7f67ccc53851bddbd052d715dd7c

Reply:
Great, thank you 👍